app/models/weet.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      userId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        field: 'user_id'
+      }
+    },
+    {
+      timestamps: true,
+      tableName: 'weets',
+      freezeTableName: true,
```

Comment:
I think `tableName` overrides `freezeTableName`, so you can remove it. What `freezeTableName` does is to force the `tableName` to be the same as the model name, which does not apply for us